### PR TITLE
Feature/login page

### LINKED ISSUE #11: Wireframe out page/dashboard layouts

### LINKED ISSUE #12: Create HTML layout for login page

### LINKED ISSUE #13: Implement Basic SASS design for login page

### PR DESCRIPTION
**Pull request by jordy1611**
### What’s this PR do?
* Sets up the login page
* User can log in as customer on manager
* CurrentUser will change to manager or current customer based on id entered
* The current user displays in the console when the login button is pressed
### Where should the reviewer start?
* index.js
* fetchAllData.js
* User.js
* Customer.js
* Manager.js
* User Customer and Manager testing files
### How should this be manually tested?
* Npm test
* Load webpage, login as customer and manager and check console
### Any background context you want to provide?
* Next step is creating dashboard for customer and manager after logged in
### What are the relevant tickets?
* Resolves #11 
* Resolves #12 
* Resolves #13

### CODE REVIEW

File: src/css/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@mixin formSizing($width, $height, $margin) {
```
Comment (kip-west) [inline src/css/_mixins.scss]: Nice use of a mixin!

File: src/css/_mixins.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
-@mixin formSizing($width, $height, $margin) {
+@mixin formSizing($width, $height, $marginLR, $marginTB) {
```
Comment (kip-west) [inline src/css/_mixins.scss:1]: This is a really cool refactor!

**Review by kip-west — COMMENTED** [review]
It seems like you put a lot of thought into your site architecture! I'm struggling with whether or not to make data repos...

File: src/index.js [hunk 3]
```diff
@@ -12,30 +12,51 @@ import BookingRepo from './Booking-repo';
...
+  console.log(password)
+  login(userName, password);
 }
 
-function validatePassword() {
-
+function login(userName, password) {
+  if (userName === 'manager' && password === 'overlook2020') {
+    loginAsManager()
+  } else if (userName.slice(0, 8) === 'customer' && password === 'overlook2020') {
```
Comment (kip-west) [inline src/index.js]: Nice work with this logic!! How do you plan on incorporating the <id> portion of the login?

File: src/Manager.js [hunk 4]
```diff
@@ -1,5 +1,15 @@
+import User from '../src/User'
 /*only 1
 porperty isManager = true (for if statements?)
 methods return todays revenue
 methods search user repo by user name, delete booking
 */
+class Manager extends User{
```
Comment (kip-west) [inline src/Manager.js:7]: Great, logical use of inheritance.

**Comment by kip-west** [comment]
Great work, Jordy!! I appreciate your clear variable names, semantic html/other accessibility considerations & skillful use of mixins.

As I was looking this over, I felt confused about the difference between your fetch methods. It seemed like you were fetching data from certain endpoints in some places, and then fetching all data later... Not sure if some of those files are leftover as you are refactoring, though.

**Review by kip-west — APPROVED** [review]

File: src/index.js (hunk 3 above)
Comment (jordy1611) [reply src/index.js]: This is how I'm going to decide if it's a manager or user logging in. So this part decides which display to make.

**Comment by jordy1611** [comment]
Yea Jake I realized that too. Went back and cleaned up a lot of my fetching. Doing it all at one time, but I still have the functionality for fetching the bookings left for when I need it later

**Pull request merged**